app/models/client.rb

```diff
@@ -1,14 +1,7 @@
 class Client < ApplicationRecord
-  belongs_to :user
-
-  has_many :language_skills, dependent: :destroy
-  accepts_nested_attributes_for :language_skills, allow_destroy: true
+  include AssociatableFields
```

Comment:
Its name is likely to change anyway, as the project grows. For the time being, I think it gives the context. Maybe it could change to AssociatedFields.